packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx

```diff
@@ -105,7 +112,9 @@ describe('NotificationDetail Page', () => {
   });
 
   test('renders NotificationDetail page without payment box if noticeCode is empty', async () => {
```

Comment:
change the text and the test with new logic
`{ recipients: [{ payment: { noticeCode: '' } }] }`
is not valid
`{ recipients: [{ payments: [] / null }] }`
is a valid case